### ISSUE TITLE
ISPG should be set to 1 when writing 3D volumes

### ISSUE THREAD

**Issue by azazellochg** [rkms86/SUSAN#3]
https://www.ccpem.ac.uk/mrc_format/mrc2014.php#note6

**Comment by rsanchezloayza** [comment]
Implemented in 5a91b4b1fb031cf354d6cd557c58ff6626b04879

**Comment by azazellochg** [comment]
Thank you. One more thing (less important, so you could ignore it):
```
>>> mrcfile.validate("average_class001.mrc")
Checking if average_class001.mrc is a valid MRC2014 file...
Error in data statistics: RMS deviation is 1.0000054836273193 but the value in the header is 0.0
```
